test/tests/performance/pbs_equiv_classes_perf.py

```diff
@@ -118,7 +118,7 @@ def test_basic(self):
         self.perf_test_result(cycle1_time, "different_equi_class", "sec")
         self.perf_test_result(cycle2_time, "single_equi_class", "sec")
         self.perf_test_result(time_diff,
-         " time_diff_bn_single_diff_equi_classes", "sec")
+                              "time_diff_bn_single_diff_equi_classes", "sec")
```

Comment:
make it as equiv_classes and equiv_class for the above 2 lines as Bhroam suggested.